src/Apps/Collect/Components/ArtworkGrid/index.tsx

```diff
@@ -38,7 +39,12 @@ class Filter extends Component<Props> {
         ? filters.state.major_periods[0]
         : filters.state[category]
 
-    return counts.map((count, index) => {
+    const options =
+      category === "major_periods"
```

Comment:
@eessex We should also remove the periodAggregation from the relay query as it'll speed up the overall query.

Here, remove any references to `periodAggregation`
https://github.com/artsy/reaction/pull/1268/files#diff-d4b3365020edfd3839ec76ed28fd0a5eR130
```javascript
     const periodAggregation = aggregations.find(
       agg => agg.slice === "MAJOR_PERIOD"
     )

```

And here, remove the `MAJOR_PERIOD` string
https://github.com/artsy/reaction/pull/1268/files#diff-d4b3365020edfd3839ec76ed28fd0a5eL269